docusaurus/docs/reference/config-types/host.v2.md

```diff
@@ -0,0 +1,291 @@
+the check results. HTTP check definitions support the following properties:
+
+* `url` - the URL to connect to.
+    * This field is required.
+* `method` - the method to use. Valid values include `GET`, `PUT`, `POST`, `PATCH`.
+    * This field is optional and defaults to `GET`.
+* `body` - the data to submit in the body of the HTTP request.
+    * This field is optional and defaults to an empty string.
+* `expectStatus` - the response status code to expect. The check will fail if a different status
+  code is encountered.
+    * This field is optional and defaults to `200`.
+* `expectInBody` - a string to expect in the status code response. The check will fail if the string
+  is not found.
+    * This field is optional. If not specified, the response body will not be checked.
+* `interval` - how often to run the health check.
+    * This field is required.
+    * Example: `5s` (5 seconds)
+    * Example: `1m` (1 minute)
+    * Example: `250ms` (250 milliseconds)
+* `timeout` - the connection timeout. Uses same format as interval.
+    * This field is required.
+    * Example: `10s` (10 seconds)
+* `actions` - how to react to health check result. Covered in more detail below.
+
+### Actions
+
+Actions define how health checks results should be reacted to. Each check may have multiple actions.
+Actions support the following properties:
+
+* `trigger` - which kind of health check result to react to. Valid values include `pass`, `fail`
+  and `change`.
+    * This field is required
+    * `change` is when the status changes from `pass` to `fail` or from `fail` to `pass`.
+* `duration` - only trigger the action if the trigger state has existed for the given duration.
+    * This field is optional. If not specified, the duration is not checked.
+    * Example: `30s` (30 seconds)
+    * Use with `change` trigger events is not recommended.
+* `consecutiveEvents` - the number of consecutive results of the given trigger type before executing
+  the action.
+    * This field is optional and defaults to 1
+    * Use with `change` trigger events is not recommended.
+* `action` - the action to take when the conditions defined by `trigger`, `duration`
+  and `consecutiveEvents` are met.
+    * This field is required
+    * Valid actions include:
+        * `mark unhealthy` - sets the associated terminator's precedence to `failed`.
+        * `mark healthy` - sets the associated terminator's precedence back from `failed` to its
+          original value.
+        * `increase cost N` - increases the cost of the associated terminator by `N`.
+        * `decrease cost N` - decreases the cost of the associated terminator by `N`.
+        * `send event` - causes a terminator event to be emitted from the controller. Useful for
+          alerting or external integrations.
+
+:::note Although multiple health checks can be configured, it's best if the actions don't overlap.
```

Comment:
put the `:::note` on its own line else it renders awkwardly. you can use this to change the title though instead of "Info" or "note" etc
![image](https://github.com/openziti/ziti-doc/assets/46322585/53c7d4e9-beb6-49c2-928b-0a689b74348c)

Reply:
fixed